word.not.found.exception=\u0421\u043B\u043E\u0432\u0430 {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
collection.not.found.exception=\u041A\u043E\u043B\u043B\u0435\u043A\u0446\u0438\u044F \u0441\u0443\u0431\u0442\u0438\u0442\u0440\u043E\u0432 {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
trial.not.found.exception=\u041F\u043E\u043F\u044B\u0442\u043A\u0430 {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
trial.finished.exception=\u041F\u043E\u043F\u044B\u0442\u043A\u0430 {0} \u0443\u0436\u0435 \u0437\u0430\u0432\u0435\u0448\u0438\u043B\u0430\u0441\u044C.
user.not.found.exception=\u041F\u043E\u043B\u044C\u0437\u043E\u0432\u0430\u0442\u044C\u0435\u043B\u044C {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
trialWord.not.found.exception=\u0420\u0435\u0437\u0443\u043B\u044C\u0442\u0430\u0442 {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
trialWord.not.free.word.exception=\u041D\u0435\u0442 \u0441\u0432\u043E\u0431\u043E\u0434\u043D\u044B\u0445 \u0441\u043B\u043E\u0432 \u0434\u043B\u044F \u041F\u043E\u043F\u044B\u0442\u043A\u0438 {0}
episode.not.found.exception=\u042D\u043F\u0438\u0437\u043E\u0434{0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.
userWord.not.found.exception=\u0421\u043B\u043E\u0432\u0430 \u043F\u043E\u043B\u044C\u0437\u043E\u0432\u0430\u0442\u0435\u043B\u044F {0} \u043D\u0435 \u0441\u0443\u0449\u0435\u0441\u0442\u0432\u0443\u0435\u0442.

api.exception.default.message=Something bad happened. Try again later.
api.exception.binding.message=\u041D\u0435\u0432\u0430\u043B\u0438\u0434\u043D\u044B\u0435 \u0432\u0445\u043E\u0434\u043D\u044B\u0435 \u0434\u0430\u043D\u043D\u044B\u0435